march_hardware/include/march_hardware/encoder/AbsoluteEncoder.h

```diff
@@ -36,16 +36,14 @@ class AbsoluteEncoder : public Encoder
   /** @brief Override comparison operator */
   friend bool operator==(const AbsoluteEncoder& lhs, const AbsoluteEncoder& rhs)
   {
-    return lhs.getSlaveIndex() == rhs.getSlaveIndex() && lhs.getTotalPositions() == rhs.getTotalPositions() &&
-           lhs.upper_soft_limit_iu_ == rhs.upper_soft_limit_iu_ &&
+    return lhs.getTotalPositions() == rhs.getTotalPositions() && lhs.upper_soft_limit_iu_ == rhs.upper_soft_limit_iu_ &&
            lhs.lower_soft_limit_iu_ == rhs.lower_soft_limit_iu_ && lhs.upper_limit_iu_ == rhs.upper_limit_iu_ &&
            lhs.lower_limit_iu_ == rhs.lower_limit_iu_ && lhs.zero_position_iu_ == rhs.zero_position_iu_;
   }
   /** @brief Override stream operator for clean printing */
   friend std::ostream& operator<<(std::ostream& os, const AbsoluteEncoder& encoder)
   {
-    return os << "slaveIndex: " << encoder.getSlaveIndex() << ", "
-              << "totalPositions: " << encoder.getTotalPositions() << ", "
+    return os << "totalPositions: " << encoder.getTotalPositions() << ", "
               << "upperHardLimit: " << encoder.upper_limit_iu_ << ", "
```

Comment:
Because the encoders no longer need a slave index in order to read and write PDOs. This now done in `Encoder` class where it accepts a `PdoSlaveInterface`.